### PR TITLE
Switch tab test

### PR DESCRIPTION
**Pull request by elenasoro**

### CODE REVIEW

File: src/test/java/tests/SauceDemoTestOnChrome.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    @Test
+    public void switchTabsTest() {
+        LoginFormPage loginFormPage = new LoginFormPage(driver);
+        LoginModel loginModel = PrepareLoginData.getStandardUserCredentials();
+        loginFormPage
+                .openLoginFormPage()
+                .inputUserName(loginModel.getLogin())
+                .inputPassword(loginModel.getPassword())
+                .clickLoginButton();
+        driver.manage().timeouts().implicitlyWait(0, TimeUnit.SECONDS);
```
Comment (khadasevich) [inline src/test/java/tests/SauceDemoTestOnChrome.java]: Let's create method to remove timeout

File: src/test/java/tests/SauceDemoTestOnChrome.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        InventoryPage inventoryPage = new InventoryPage(driver);
+        inventoryPage.goToLinkedin();
+        ArrayList<String> tabs2 = new ArrayList<String> (driver.getWindowHandles());
+        driver.switchTo().window(tabs2.get(1));
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[data-test-id=nav-logo]")));
```
Comment (khadasevich) [inline src/test/java/tests/SauceDemoTestOnChrome.java]: Let's move this action to page

File: src/test/java/tests/SauceDemoTestOnChrome.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        Assert.assertTrue(driver.findElement(By.cssSelector("[data-test-id=nav-logo]")).isDisplayed(), "Linkedin Logo not displayed");
```
Comment (khadasevich) [inline src/test/java/tests/SauceDemoTestOnChrome.java]: Please move element to page

**Review by khadasevich — COMMENTED** [review]
Global points: left one Base test and do not use binary to start Safari

File: src/test/java/tests/SauceDemoTestOnChrome.java (hunk 1 above)
Comment (elenasoro) [reply src/test/java/tests/SauceDemoTestOnChrome.java]: done

File: src/test/java/tests/SauceDemoTestOnChrome.java (hunk 2 above)
Comment (elenasoro) [reply src/test/java/tests/SauceDemoTestOnChrome.java]: done

File: src/test/java/tests/SauceDemoTestOnChrome.java (hunk 3 above)
Comment (elenasoro) [reply src/test/java/tests/SauceDemoTestOnChrome.java]: done

**Review by khadasevich — APPROVED** [review]

**Pull request merged**